otx/algorithms/classification/adapters/mmcls/data/pipelines.py

```diff
@@ -62,3 +64,76 @@ def __call__(self, results: Dict[str, Any]):
             results["img"] = results["img"].astype(np.float32)
 
         return results
+
+
+@PIPELINES.register_module()
+class RandomAppliedTrans:
+    """Randomly applied transformations.
+
+    :param transforms: List of transformations in dictionaries
+    :param p: Probability, defaults to 0.5
+    """
+
+    def __init__(self, transforms: List, p: float = 0.5):
+        t = [build_from_cfg(t, PIPELINES) for t in transforms]  # pylint: disable=invalid-name
+        self.trans = T.RandomApply(t, p=p)
+
+    @check_input_parameters_type()
+    def __call__(self, results: Dict[str, Any]):
+        """Callback function of RandomAppliedTrans.
+
+        :param results: Inputs to be transformed.
+        """
+        return self.trans(results)
+
+    def __repr__(self):
+        """Set repr of RandomAppliedTrans."""
+        repr_str = self.__class__.__name__
+        return repr_str
+
+
+@PIPELINES.register_module
+class OTXColorJitter(T.ColorJitter):
+    """Wrapper for ColorJitter in torchvision.transforms.
+
+    Use this instead of mmcls's because there is no `hue` parameter in mmcls ColorJitter.
+    """
+
+    def __call__(self, results):
+        """Callback function of OTXColorJitter.
+
+        :param results: Inputs to be transformed.
+        """
+        results["img"] = np.array(self.forward(Image.fromarray(results["img"])))
```

Comment:
AFAIK, we agreed to avoid this kind of duplication (numpy --> PIL --> numpy). do you have a plan to avoid it?

Reply:
It also remains on segmentation task. As @jaegukhyun said, It would be better to be progressed with another PR.

Reply:
Why do we move on to another PR for the refactoring task? @jaegukhyun

Reply:
I don't think how they are refactored.
I think we can add PR as 'optimize data pipelines inherited from torchvision' and discuss it there.

Reply:
As we have a limited time for feature freeze, and this refactoring task will be changed after merging segmentation. 
I think it would be better to discuss how we can make a better code after merging new feature PRs(self-sl-cls, self-sl-seg).
However, if you think it is necessary changes, then it should be handled in this PR

Reply:
Didn't we have implemented a function to change this line? I'm aware of there was implemented function. If it didn't exist, I agreed with your opinions. 

+ just wondering: Self-SL cls/seg feature is included in OTXv1.0 feature freeze?

Reply:
Agree with jaeguk. Adding to this, it seems like only pipelines using for self-sl just registered here, while others in mpa transform files. I think we better discuss later.